content/soarian/dstu2/individuals/patient.md

```diff
@@ -38,10 +38,10 @@ The following extensions can also be returned:
 
 	GET /Patient?:parameters
 	
-_Implementation Notes_
+### Implementation Notes
 
 * The API returns the most current patient list.
-* The value for gender in the patient resource may or may not represent the value of sex explicitly recorded at birth.  ONC has clarified that birth sex is the CCDS (common clinical data set) value.  This information, if charted and configured is returned as part of the Observation resource.
+* This is the Administrative Gender.  Birth Sex,  if charted and configured, is returned as part of the Observation resource.   
```

Comment:
Should be "Returns the administrative gender only. Birth Sex, if charted and configured, is returned from the Observation resource.